jobs/src/main/scala/dcos/metronome/model/JobSpec.scala

```diff
@@ -4,8 +4,12 @@ package model
 import com.wix.accord.Validator
 import com.wix.accord.dsl._
 import dcos.metronome.model.JobRunSpec._
+import dcos.metronome.utils.glue.MarathonConversions
 import mesosphere.marathon.api.v2.Validation._
-import mesosphere.marathon.plugin.{ ApplicationSpec, EnvVarValue, NetworkSpec, Secret, VolumeMountSpec, VolumeSpec }
+import mesosphere.marathon
+import mesosphere.marathon.plugin.{ ApplicationSpec, NetworkSpec, Secret, VolumeSpec, VolumeMountSpec }
+
+import scala.collection.immutable._
```

Comment:
is there a reason this is necessary?   the double package metronome object should use immutable for Seq and IndexedSeq